src/web/views/check.njk

```diff
@@ -1,85 +1,31 @@
 {% extends "templates/form.njk" %}
 {% from "input/macro.njk" import govukInput %}
 {% from "table/macro.njk" import govukTable %}
+{% from "fieldset/macro.njk" import govukFieldset %}
 
-{% set buttonText = "Confirm" %}
+{% set safeRows = [] %}
+{% for heading, content in checkRowData %}
+  {% set row = [{ text: heading.text }, { text: content.text | safe } ] %}
```

Comment:
Is there any situation where the `| safe` filter here could be a security vulnerability?

Reply:
Possibly, but I think it should be filtered on the input sanitisation rather than the output here. If you have another solution which will work with Nunjucks then please let me know.